pages/services/kubernetes/2.0.0-1.12.1/operations/exposing-the-kubernetes-api/index.md

```diff
@@ -307,7 +307,7 @@ frontend frontend_all
     use_backend backend_devkubernetes02 if { ssl_fc_sni devkubernetes02.example.com }
 ```
 
-Notice how `backend_devkubernetes01` and `backend_devkubernetes02` match the names of the backends defined in the first step.
+Notice how `backend_devkubernetes01` and `backend_devkubernetes02` match the the names of the backends defined in the first step.
```

Comment:
I see a few typos introduced like this? almost like they are being undone.

Reply:
Thanks @alejandroEsc !